KMaina/Fast-Food-App
# #160219327 Creating the sign up page

#### What does this PR do?
Allow a user to create an account with the site if they do not have one.
#### Description of Task to be completed?
The user should fill in a form with

- A username

- An email address

- A password

- Confirm the same password they chose to check if they are the same

- Upload an image to be used as a profile picture

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run` cd Fast-Food-App`

- run `git checkout -b ft-sign-up-page-#160219327`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-sign-up-page-#160219327`

- In the folder where you cloned the repo, you should open the UI folder.

- Open the sign-up.html to see it in your browser for review.

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160219327
